src/main/scala/io/joern/typestubs/Main.scala

```diff
@@ -1,8 +1,11 @@
 package io.joern.typestubs
 
+import io.joern.typestubs.ruby.BuiltinPackageDownloader
+
 /** Example program that makes use of Joern as a library */
 object Main {
-
-  def main(args: Array[String]): Unit = {}
-
+  def main(args: Array[String]): Unit = {
```

Comment:
In an upcoming or issue, it would be good to be able to specify if we want uncompressed raw JSON (useful for debugging) or the compressed/mpk output

Reply:
Also, potentially an argument to limit scraping/type stub generation to one language, where default is all

Reply:
Will do this in the next PR